scripts/lib/setup_venv.py

```diff
@@ -55,6 +59,8 @@
     "postgresql-libs",  # libpq-dev on apt
     "openssl-devel",
     "jq",
+    # We may or may not need it here.
+    "libsasl2-dev",
```

Comment:
`libsasl2-dev` is definitely not a CentOS or Fedora package name.

Reply:
Okay, checked!